src/pkg/caendr/caendr/models/datastore/trait_file.py

```diff
@@ -74,6 +74,8 @@ def bucket(self):
 
   @property
   def prefix(self):
+    if self.dataset == 'public':
```

Comment:
try to avoid string literals for any comparisons used for business logic. Instead, prefer comparing using enumerators: 
```python
class DatasetMode(Enum):
    PUBLIC = 1
    NOT_PUBLIC = 2
    NOT_NOT_PUBLIC = 3
```

Then use it as:
```python
  if self.dataset == DatasetMode.PUBLIC
```

Reply:
Pushed in the commit d94bf6173d7cfda13e8224c7badddbdc7ae7ff9b